models/migrations/migrations.go

```diff
@@ -57,7 +57,7 @@ type Version struct {
 // update minDBVersion accordingly
 var migrations = []Migration{
 
-	// Gitea 1.5.3 ends at v70
```

Comment:
It seems these changes are unrelated?

Reply:
they are - should I move it into it's own pull?

Reply:
moved into  #13789